packages/mirage-bootvar-xen/mirage-bootvar-xen.0.5.0/opam

```diff
@@ -0,0 +1,45 @@
+opam-version: "2.0"
+name: "mirage-bootvar-xen"
+maintainer: "Magnus Skjegstad <magnus@skjegstad.com>"
+authors: "Magnus Skjegstad <magnus@skjegstad.com>"
+homepage: "https://github.com/mirage/mirage-bootvar-xen"
+bug-reports: "https://github.com/mirage/mirage-bootvar-xen/issues/"
+dev-repo: "git+https://github.com/TheLortex/mirage-bootvar-xen.git"
+license: "ISC"
+
+build: [
+  [ "jbuilder" "subst" "-p" name ] {pinned}
```

Comment:
jbuilder?? :-)

Reply:
Looks like mirage-bootvar-xen is still using jbuild files !